monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -63,8 +63,8 @@ import scala.util.{Failure, Success, Try}
   * [[monix.eval.Task.apply Task.apply]]:
   *
   * {{{
-  *   val hello = Task.eval("Hello ")
-  *   val world = Task("World!")
+  *   scala> val hello = Task.eval("Hello ")
```

Comment:
I don’t really like having a Scala prompt in code samples. I know it has become a more common practice since “tut”, but code samples need to be copy/paste-able.

I also hope this doesn’t yield any extra output. I’m guessing that it doesn’t.

Can we not do the `scala>` thing?

Reply:
I'm also not a big fan of it, but unfortunately, this is how sbt-doctest works. 
to my understanding, it's can be either

```
 * >>> Test.f(
 * ...   10
 * ... )
 * 20
 ```
or 
```
 * scala> Test.f(
 *      |   20
 *      | )
``` 

I'm not sure how much of an effort would be to change it to work without `scala>`, so I will check it and get back

Reply:
Maybe we should open an issue on the sbt-doctest repository.